Vue: Lint command has 113 errors

```
npm run lint

> vueapp@0.0.0 lint
> eslint --ext .js,.ts,.vue src


/private/tmp/jhlite/vueapp/src/main/webapp/app/common/domain/Logger.ts
  1:25  error  Replace `'@/common/domain/Message'` with `"@/common/domain/Message"`  prettier/prettier

/private/tmp/jhlite/vueapp/src/main/webapp/app/common/primary/app/App.component.ts
  2:9  error  Replace `'AppVue'` with `"AppVue"`  prettier/prettier

/private/tmp/jhlite/vueapp/src/main/webapp/app/common/primary/app/index.ts
  1:26  error  Replace `'./App.component'` with `"./App.component"`  prettier/prettier
  2:20  error  Replace `'./AppVue.vue'` with `"./AppVue.vue"`        prettier/prettier

/private/tmp/jhlite/vueapp/src/main/webapp/app/common/primary/homepage/Homepage.component.ts
  2:9   error  Replace `'HomepageVue'` with `"HomepageVue"`  prettier/prettier
  5:16  error  Replace `'vueapp'` with `"vueapp"`            prettier/prettier

/private/tmp/jhlite/vueapp/src/main/webapp/app/common/primary/homepage/index.ts
  1:31  error  Replace `'./Homepage.component'` with `"./Homepage.component"`  prettier/prettier
  2:25  error  Replace `'./HomepageVue.vue'` with `"./HomepageVue.vue"`        prettier/prettier

/private/tmp/jhlite/vueapp/src/main/webapp/app/common/secondary/ConsoleLogger.ts
  1:24  error  Replace `'@/common/domain/Logger'` with `"@/common/domain/Logger"`    prettier/prettier
  2:25  error  Replace `'@/common/domain/Message'` with `"@/common/domain/Message"`  prettier/prettier

/private/tmp/jhlite/vueapp/src/main/webapp/app/env.d.ts
  3:16  error  Replace `'*.vue'` with `"*.vue"`  prettier/prettier
  4:35  error  Replace `'vue'` with `"vue"`      prettier/prettier

/private/tmp/jhlite/vueapp/src/main/webapp/app/http/AxiosHttp.ts
   1:66  error  Replace `'axios'` with `"axios"`                                                                                                                     prettier/prettier
  12:38  error  Replace `uri:·string,·data?:·Payload` with `⏎····uri:·string,⏎····data?:·Payload,⏎··`                                                                prettier/prettier
  16:39  error  Replace `uri:·string,·data?:·Payload,·config?:·AxiosRequestConfig` with `⏎····uri:·string,⏎····data?:·Payload,⏎····config?:·AxiosRequestConfig,⏎··`  prettier/prettier

/private/tmp/jhlite/vueapp/src/main/webapp/app/main.ts
   1:27  error  Replace `'vue'` with `"vue"`                                    prettier/prettier
   2:24  error  Replace `'./common/primary/app'` with `"./common/primary/app"`  prettier/prettier
   3:20  error  Replace `'./router/router'` with `"./router/router"`            prettier/prettier
   4:29  error  Replace `'pinia'` with `"pinia"`                                prettier/prettier
   5:26  error  Replace `'pinia-plugin-persist'` with `"pinia-plugin-persist"`  prettier/prettier
  16:11  error  Replace `'#app'` with `"#app"`                                  prettier/prettier

/private/tmp/jhlite/vueapp/src/main/webapp/app/router/router.ts
   1:29  error  Replace `'@/common/primary/homepage'` with `"@/common/primary/homepage"`  prettier/prettier
   2:48  error  Replace `'vue-router'` with `"vue-router"`                                prettier/prettier
   6:11  error  Replace `'/'` with `"/"`                                                  prettier/prettier
   7:23  error  Replace `'Homepage'` with `"Homepage"`                                    prettier/prettier
  10:11  error  Replace `'/home'` with `"/home"`                                          prettier/prettier
  11:11  error  Replace `'Homepage'` with `"Homepage"`                                    prettier/prettier

/private/tmp/jhlite/vueapp/src/main/webapp/app/vue/VueProp.ts
  1:26  error  Replace `'vue'` with `"vue"`  prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/common/domain/Logger.fixture.ts
  1:34  error  Replace `'sinon'` with `"sinon"`                                    prettier/prettier
  2:24  error  Replace `'@/common/domain/Logger'` with `"@/common/domain/Logger"`  prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/common/primary/app/App.spec.ts
   1:38  error  Replace `'vitest'` with `"vitest"`                                          prettier/prettier
   2:42  error  Replace `'@vue/test-utils'` with `"@vue/test-utils"`                        prettier/prettier
   3:24  error  Replace `'@/common/primary/app'` with `"@/common/primary/app"`              prettier/prettier
  10:15  error  Replace `'router-link',·'router-view'` with `"router-link",·"router-view"`  prettier/prettier
  15:10  error  Replace `'App'` with `"App"`                                                prettier/prettier
  16:6   error  Replace `'should·exist'` with `"should·exist"`                              prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/common/primary/homepage/Homepage.spec.ts
   1:38  error  Replace `'vitest'` with `"vitest"`                                        prettier/prettier
   2:42  error  Replace `'@vue/test-utils'` with `"@vue/test-utils"`                      prettier/prettier
   3:29  error  Replace `'@/common/primary/homepage'` with `"@/common/primary/homepage"`  prettier/prettier
  11:10  error  Replace `'App'` with `"App"`                                              prettier/prettier
  12:6   error  Replace `'should·exist'` with `"should·exist"`                            prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/common/secondary/ConsoleLogger.spec.ts
   1:42  error  Replace `'vitest'` with `"vitest"`                                                      prettier/prettier
   3:27  error  Replace `'@/common/secondary/ConsoleLogger'` with `"@/common/secondary/ConsoleLogger"`  prettier/prettier
   5:10  error  Replace `'ConsoleLogger'` with `"ConsoleLogger"`                                        prettier/prettier
   6:6   error  Replace `'should·log·an·error'` with `"should·log·an·error"`                            prettier/prettier
  11:29  error  Replace `'Error·message'` with `"Error·message"`                                        prettier/prettier
  13:25  error  Replace `'An·error·occurs'` with `"An·error·occurs"`                                    prettier/prettier
  16:41  error  Replace `'An·error·occurs\n'` with `"An·error·occurs\n"`                                prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/http/AxiosHttp.spec.ts
    1:38  error  Replace `'vitest'` with `"vitest"`                                                                                                                   prettier/prettier
    2:31  error  Replace `'axios'` with `"axios"`                                                                                                                     prettier/prettier
    3:27  error  Replace `'@/http/AxiosHttp'` with `"@/http/AxiosHttp"`                                                                                               prettier/prettier
    4:54  error  Replace `'./AxiosStub'` with `"./AxiosStub"`                                                                                                         prettier/prettier
   11:12  error  Replace `'content'` with `"content"`                                                                                                                 prettier/prettier
   19:11  error  Replace `'content'` with `"content"`                                                                                                                 prettier/prettier
   26:20  error  Replace `'/uri'` with `"/uri"`                                                                                                                       prettier/prettier
   29:38  error  Replace `uri:·string,·payload:·Payload,·result:·AxiosResponse<Result>` with `⏎··uri:·string,⏎··payload:·Payload,⏎··result:·AxiosResponse<Result>,⏎`  prettier/prettier
   34:10  error  Replace `'axiosHttp'` with `"axiosHttp"`                                                                                                             prettier/prettier
   35:12  error  Replace `'GET'` with `"GET"`                                                                                                                         prettier/prettier
   36:8   error  Replace `'should·get·content'` with `"should·get·content"`                                                                                           prettier/prettier
   41:50  error  Replace `'/uri'` with `"/uri"`                                                                                                                       prettier/prettier
   47:8   error  Replace `'should·get·content·with·params'` with `"should·get·content·with·params"`                                                                   prettier/prettier
   52:35  error  Replace `'/uri',·{·params:·{·beer:·'chips'` with `"/uri",·{·params:·{·beer:·"chips"`                                                                 prettier/prettier
   55:39  error  Replace `'chips'` with `"chips"`                                                                                                                     prettier/prettier
   59:12  error  Replace `'PUT'` with `"PUT"`                                                                                                                         prettier/prettier
   60:8   error  Replace `'should·only·get·content'` with `"should·only·get·content"`                                                                                 prettier/prettier
   65:50  error  Replace `'/uri'` with `"/uri"`                                                                                                                       prettier/prettier
   71:8   error  Replace `'should·send·and·get·content'` with `"should·send·and·get·content"`                                                                         prettier/prettier
   76:59  error  Replace `'/uri',·fakePayload()` with `⏎········"/uri",⏎········fakePayload(),⏎······`                                                                prettier/prettier
   83:12  error  Replace `'POST'` with `"POST"`                                                                                                                       prettier/prettier
   84:8   error  Replace `'should·only·get·content'` with `"should·only·get·content"`                                                                                 prettier/prettier
   89:51  error  Replace `'/uri'` with `"/uri"`                                                                                                                       prettier/prettier
   95:8   error  Replace `'should·send·and·get·content'` with `"should·send·and·get·content"`                                                                         prettier/prettier
  100:60  error  Replace `'/uri',·fakePayload()` with `⏎········"/uri",⏎········fakePayload(),⏎······`                                                                prettier/prettier
  107:12  error  Replace `'DELETE'` with `"DELETE"`                                                                                                                   prettier/prettier
  108:8   error  Replace `'should·get·content'` with `"should·get·content"`                                                                                           prettier/prettier
  113:53  error  Replace `'/uri'` with `"/uri"`                                                                                                                       prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/http/AxiosHttpStub.ts
   1:34  error  Replace `'sinon'` with `"sinon"`                                      prettier/prettier
   2:46  error  Replace `'@/http/AxiosHttp'` with `"@/http/AxiosHttp"`                prettier/prettier
  17:4   error  Replace `·as·AxiosHttpStub)` with `)·as·AxiosHttpStub`                prettier/prettier
  22:4   error  Replace `·as·AxiosHttpResponse<T>)` with `)·as·AxiosHttpResponse<T>`  prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/http/AxiosStub.ts
   1:46  error  Replace `'axios'` with `"axios"`                                prettier/prettier
   2:34  error  Replace `'sinon'` with `"sinon"`                                prettier/prettier
  17:4   error  Replace `·as·AxiosStubInstance)` with `)·as·AxiosStubInstance`  prettier/prettier
  22:4   error  Replace `·as·AxiosResponse<T>)` with `)·as·AxiosResponse<T>`    prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/router/Router.spec.ts
   1:38  error  Replace `'vitest'` with `"vitest"`                                                        prettier/prettier
   2:42  error  Replace `'@vue/test-utils'` with `"@vue/test-utils"`                                      prettier/prettier
   3:24  error  Replace `'@/common/primary/app'` with `"@/common/primary/app"`                            prettier/prettier
   5:20  error  Replace `'@/router/router'` with `"@/router/router"`                                      prettier/prettier
  12:15  error  Replace `'router-link',·'router-view'` with `"router-link",·"router-view"`                prettier/prettier
  18:10  error  Replace `'Router'` with `"Router"`                                                        prettier/prettier
  19:6   error  Replace `'Should·redirect·to·App·by·default'` with `"Should·redirect·to·App·by·default"`  prettier/prettier
  21:23  error  Replace `'/'` with `"/"`                                                                  prettier/prettier
  27:6   error  Replace `'Should·go·to·AppVue'` with `"Should·go·to·AppVue"`                              prettier/prettier
  29:23  error  Replace `'/app'` with `"/app"`                                                            prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/vue/vue-prop/ArrayComponentVue.vue
   6:33  error  Replace `'vue'` with `"vue"`                              prettier/prettier
   7:27  error  Replace `'@/vue/VueProp'` with `"@/vue/VueProp"`          prettier/prettier
  10:9   error  Replace `'ArrayComponentVue'` with `"ArrayComponentVue"`  prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/vue/vue-prop/ObjectComponentVue.vue
   6:33  error  Replace `'vue'` with `"vue"`                                prettier/prettier
   7:28  error  Replace `'@/vue/VueProp'` with `"@/vue/VueProp"`            prettier/prettier
  15:9   error  Replace `'ObjectComponentVue'` with `"ObjectComponentVue"`  prettier/prettier

/private/tmp/jhlite/vueapp/src/test/javascript/spec/vue/vue-prop/VueProp.spec.ts
   1:34  error  Replace `'vitest'` with `"vitest"`                                      prettier/prettier
   2:30  error  Replace `'@vue/test-utils'` with `"@vue/test-utils"`                    prettier/prettier
   4:31  error  Replace `'./ArrayComponentVue.vue'` with `"./ArrayComponentVue.vue"`    prettier/prettier
   5:32  error  Replace `'./ObjectComponentVue.vue'` with `"./ObjectComponentVue.vue"`  prettier/prettier
   6:10  error  Replace `'VueProp'` with `"VueProp"`                                    prettier/prettier
   7:12  error  Replace `'Array'` with `"Array"`                                        prettier/prettier
   8:8   error  Replace `'Should·type'` with `"Should·type"`                            prettier/prettier
  11:21  error  Replace `'first',·'second'` with `"first",·"second"`                    prettier/prettier
  15:43  error  Replace `'first',·'second'` with `"first",·"second"`                    prettier/prettier
  19:12  error  Replace `'Object'` with `"Object"`                                      prettier/prettier
  20:8   error  Replace `'Should·type'` with `"Should·type"`                            prettier/prettier
  24:25  error  Replace `'Some·string'` with `"Some·string"`                            prettier/prettier
  32:44  error  Replace `'Some·string'` with `"Some·string"`                            prettier/prettier

✖ 113 problems (113 errors, 0 warnings)
  113 errors and 0 warnings potentially fixable with the `--fix` option.
```

--- a/src/main/resources/generator/client/vue/.eslintrc.js
+++ b/src/main/resources/generator/client/vue/.eslintrc.js
@@ -9,6 +9,6 @@ module.exports = {
   rules: {
     quotes: ['error', 'single', { avoidEscape: true }],
     '@typescript-eslint/no-unused-vars': ['error'],
-    'prettier/prettier': ['error'],
+    'prettier/prettier': ['error', { singleQuote: true }],
   },
 };

--- a/src/main/resources/generator/client/vue/test/spec/http/AxiosHttp.spec.ts.mustache
+++ b/src/main/resources/generator/client/vue/test/spec/http/AxiosHttp.spec.ts.mustache
@@ -26,7 +26,11 @@ const expectForQuerying = (uri: string, result: AxiosResponse<Result>) => {
   expect(uri).toBe('/uri');
 };
 
-const expectForSendingAndQuerying = (uri: string, payload: Payload, result: AxiosResponse<Result>) => {
+const expectForSendingAndQuerying = (
+  uri: string,
+  payload: Payload,
+  result: AxiosResponse<Result>,
+) => {
   expect(payload).toEqual<Payload>(fakePayload());
   expectForQuerying(uri, result);
 };
@@ -73,7 +77,10 @@ describe('axiosHttp', () => {
       axiosInstance.put.resolves(responseResult());
       const axiosHttp = new AxiosHttp(axiosInstance);
 
-      const result = await axiosHttp.put<Result, Payload>('/uri', fakePayload());
+      const result = await axiosHttp.put<Result, Payload>(
+        '/uri',
+        fakePayload(),
+      );
 
       const [uri, payload] = axiosInstance.put.getCall(0).args;
       expectForSendingAndQuerying(uri, payload, result);
@@ -97,7 +104,10 @@ describe('axiosHttp', () => {
       axiosInstance.post.resolves(responseResult());
       const axiosHttp = new AxiosHttp(axiosInstance);
 
-      const result = await axiosHttp.post<Result, Payload>('/uri', fakePayload());
+      const result = await axiosHttp.post<Result, Payload>(
+        '/uri',
+        fakePayload(),
+      );
 
       const [uri, payload] = axiosInstance.post.getCall(0).args;
       expectForSendingAndQuerying(uri, payload, result);

--- a/src/main/resources/generator/client/vue/test/spec/http/AxiosHttpStub.ts.mustache
+++ b/src/main/resources/generator/client/vue/test/spec/http/AxiosHttpStub.ts.mustache
@@ -14,9 +14,9 @@ export const stubAxiosHttp = (): AxiosHttpStub =>
     post: sinon.stub(),
     delete: sinon.stub(),
     put: sinon.stub(),
-  } as AxiosHttpStub);
+  }) as AxiosHttpStub;
 
 export const dataBackendResponse = <T>(data: T): AxiosHttpResponse<T> =>
   ({
     data,
-  } as AxiosHttpResponse<T>);
+  }) as AxiosHttpResponse<T>;

--- a/src/main/resources/generator/client/vue/test/spec/http/AxiosStub.ts.mustache
+++ b/src/main/resources/generator/client/vue/test/spec/http/AxiosStub.ts.mustache
@@ -14,9 +14,9 @@ export const stubAxiosInstance = (): AxiosStubInstance =>
     put: sinon.stub(),
     post: sinon.stub(),
     delete: sinon.stub(),
-  } as AxiosStubInstance);
+  }) as AxiosStubInstance;
 
 export const dataAxiosResponse = <T>(data: T): AxiosResponse<T> =>
   ({
     data,
-  } as AxiosResponse<T>);
+  }) as AxiosResponse<T>;

--- a/src/main/resources/generator/client/vue/webapp/app/http/AxiosHttp.ts.mustache
+++ b/src/main/resources/generator/client/vue/webapp/app/http/AxiosHttp.ts.mustache
@@ -9,11 +9,18 @@ export class AxiosHttp {
     return this.axiosInstance.get<Result>(uri, config);
   }
 
-  async put<Result, Payload = never>(uri: string, data?: Payload): Promise<AxiosResponse<Result>> {
+  async put<Result, Payload = never>(
+    uri: string,
+    data?: Payload,
+  ): Promise<AxiosResponse<Result>> {
     return this.axiosInstance.put<Result>(uri, data);
   }
 
-  async post<Result, Payload = never>(uri: string, data?: Payload, config?: AxiosRequestConfig): Promise<AxiosResponse<Result>> {
+  async post<Result, Payload = never>(
+    uri: string,
+    data?: Payload,
+    config?: AxiosRequestConfig,
+  ): Promise<AxiosResponse<Result>> {
     return this.axiosInstance.post<Result>(uri, data, config);
   }
 

--- a/src/test/javascript/spec/http/AxiosHttp.spec.ts
+++ b/src/test/javascript/spec/http/AxiosHttp.spec.ts
@@ -1,7 +1,7 @@
+import { describe, it, expect } from 'vitest';
 import { AxiosResponse } from 'axios';
 import { AxiosHttp } from '@/http/AxiosHttp';
 import { dataAxiosResponse, stubAxiosInstance } from './AxiosStub';
-import { describe, it, expect } from 'vitest';
 
 interface Payload {
   payload: string;

